packaging/rpm/microshift.spec

```diff
@@ -221,6 +221,24 @@ The microshift-gateway-api-release-info package provides release information fil
 release. These files contain the list of container image references used by Gateway API
 and can be used to embed those images into osbuilder blueprints.
 
+%package ai-model-serving
+Summary: AI Model Serving for MicroShift
+ExclusiveArch: x86_64
+Requires: microshift = %{version}
+
+%description ai-model-serving
+The microshift-ai-model-serving package provides manifests for the RHOAI based kserve and serving runtimes.
```

Comment:
Please, work with docs team on the description line. "kserve and serving runtimes" seems to be a bit cryptic

Reply:
Changed it to be less cryptic. Given our vast documentation, I doubt we need to go into much detail in RPM descriptions.